File: 04) PHP Web Development Basics/10) OOP Basics - Exercises/6. Raw Data/Tire.php
<?php

namespace Van\Classes;

class Tire
{
    /**
     * @var int
     */
    private $age;

    /**
     * @var float
     */
    private $pressure;

    /**
     * Tire constructor.
     *
     * @param int $age
     * @param float $pressure
     */
    public function __construct(int $age, float $pressure)
    {
        $this->age = $age;
        $this->pressure = $pressure;
    }

    /**
     * @return int
     */
    public function getAge(): int
    {
        return $this->age;
    }

    /**
     * @param int $age
     */
    public function setAge(int $age)
    {
        $this->age = $age;
    }

    /**
     * @return float
     */
    public function getPressure(): float
    {
        return $this->pressure;
    }

    /**
     * @param float $pressure
     */
    public function setPressure(float $pressure)
    {
        $this->pressure = $pressure;
    }
}

File: 09) Java Advanced - January 2019/14. Defining Classes - Exercises/04. Raw Data/CarParts/Tire.java
package CarParts;

public class Tire {

    private Integer age;

    private Double pressure;

    public Tire(Integer age, Double pressure) {
        setAge(age);
        setPressure(pressure);
    }

    public Integer getAge() {
        return age;
    }

    private void setAge(Integer age) {
        this.age = age;
    }

    public Double getPressure() {
        return pressure;
    }

    private void setPressure(Double pressure) {
        this.pressure = pressure;
    }
}
